runtest.sh

```diff
@@ -86,7 +86,7 @@ docker run \
 
 echo "Starting test suite"
 docker run -t \
-  -v "${here}:/f8a_worker:ro,Z" \
+  -v "${here}:/f8a_worker:rw,Z" \
```

Comment:
This is unfortunate, but I don't have any better solution, so I think I can live with that if it's just for tests.
Maybe if you could put a comment there, explaining it ?